tests/query_runner/test_trino.py

```diff
@@ -0,0 +1,59 @@
+"""
+Some test cases for Trino.
+"""
+import re
+from unittest import TestCase
+from unittest.mock import patch
+
+from redash.query_runner.trino import Trino
+
+
+class TestTrino(TestCase):
+    catalog_name = "memory"
+    schema_name = "default"
+    table_name = "users"
+    column_name = "id"
+    column_type = "integer"
+
+    @staticmethod
+    def side_effect_run_query(query, user):
+        catalog_query_pattern = r"\s*SHOW\s+CATALOGS\s*"
+        schema_query_pattern = r"""
+        \s*SELECT\s+table_schema,\s+table_name,\s+column_name,\s+data_type
+        \s+FROM\s+(\w+\.)?information_schema\.columns
+        \s+WHERE\s+table_schema\s+NOT\s+IN\s+\('pg_catalog',\s+'information_schema'\)\s*
```

Comment:
I think that instead of hardcoding the queries that get run in the runner here, we could patch the `.run_query` method to return the results in L27 / L29-31 directly. That way we don't depend on the exact implementation of the Trino runner, but rather just on the input/output being exactly the same every time.

Reply:
It is addressed in 33d30c6. What do you think about that change @guidopetri ?